third_party/txt/src/txt/platform_linux.cc

```diff
@@ -13,14 +13,40 @@
 namespace txt {
 
 std::vector<std::string> GetDefaultFontFamilies() {
-  return {"SamsungOneUI", "BreezeSans", "Ubuntu", "Cantarell", "DejaVu Sans", "Liberation Sans", "Arial"};
+  return {
+      "SamsungOneUI",           "SamsungOneUIArabic",
```

Comment:
Where did you get these lists? wearable devices?

Reply:
Wearable and TV devices. 

Actually, @swift-kim mentioned that these fonts are not enough(need to add fonts in emulators), so this PR still needs update.